docs/book/src/reference/compiler_intrinsics.md

```diff
@@ -372,3 +372,37 @@ Runtime bound checks are not generated, and must be done manually when and where
 
 - `item` is a reference to an array or a reference to a slice;
 - when `index` is a literal, it must be smaller than `item` length;
+
+---
+
+```sway
+__dbg(value: T) -> T
```

Comment:
As addition to the discussion on typechecking the `__dbg` argument, shouldn't the signature be:
```
__dbg<T>(value: T) -> T where T: Debug
```
(Noticed now that `<T>` is missing in the docs :smile:)

Reply:
Done. I also fixed other missing places.